docs/flow/unidist/core/backends/mpi/core/serialization.rst

```diff
@@ -8,13 +8,21 @@
 Serialization API
 """""""""""""""""
 
-Class :py:class:`~unidist.core.backends.mpi.core.serialization.MPISerializer` supports data serialization of complex types
-with lambdas, functions, member functions. Moreover, the class support out-of-band data serialization
+Class :py:class:`~unidist.core.backends.mpi.core.serialization.ComplexDataSerializer` supports data serialization of complex types
+with lambdas, functions, member functions and large data arrays. In particular, the class supports out-of-band data serialization
 for a set of pickle 5 protocol compatible libraries - pandas and NumPy, specifically ``pandas.DataFrame``, ``pandas.Series`` and ``np.ndarray`` types.
+Use this class in case serialization of compound objects with different unknown types, poissibly large arrays.
+Class :py:class:`~unidist.core.backends.mpi.core.serialization.SimpleDataSerializer` supports data serialization of simple data types.
+Also, the class has an API for lambdas, functions, member functions serialization (and uses cloudpickle library for that purpose),
+but with no performance optimization for large raw data.  Use this class when you know exact object type for serialization and it`s not cosists of large datasets.
```

Comment:
```suggestion
This class is used in case of serialization of compound objects with different unknown types, possibly large arrays.
Class :py:class:`~unidist.core.backends.mpi.core.serialization.SimpleDataSerializer` supports data serialization of simple data types.
Also, the class has an API for serialization of lambdas, functions, member functions (and uses ``cloudpickle`` library for that purpose),
but with no performance optimization for large raw data. This class is used when exact object type is known for serialization and it doesn't consist of large datasets.
```

Reply:
Also, check the links to `SimpleDataSerializer` and `ComplexDataSerializer` please. These do not work. Btw, API section is empty as well.